src/worker/lambda/packages/packagePuller.go

```diff
@@ -36,8 +38,10 @@ type Package struct {
 	Meta         PackageMeta
 	installMutex sync.Mutex
 	installed    uint32
+	Size int
```

Comment:
Will need to be protected by some lock.  Perhaps with installMutex?  Which would then need to be renamed to reflect the broader usage